source/_integrations/growatt_server.markdown

```diff
@@ -17,13 +17,13 @@ This will log into your Growatt account and grab the first "Plant", after which
 
 ## Configuration
 
-Add the following to your `configuration.yaml` file:
+You can set this integration up using the integration page.
```

Comment:
Please describe to the end-user how to do that (e.g., where to go, click and such). See other integrations for inspiration.